beancount/utils/misc_utils.py

```diff
@@ -282,15 +281,15 @@ def compute_unique_clean_ids(strings):
     return idmap
 
 
-def escape_string(string: str) -> str:
-    """Escape backslashes, double quotes, and control characters.
+def escape_string(string):
+    """Escape quotes and backslashes in payee and narration.
```

Comment:
I don't get this docstring change. Where is this function checking that it is being applied to payee and narration?

Reply:
this is a revert, it's the old docstring.